data/huion-h420.tablet

```diff
@@ -0,0 +1,23 @@
+# HUION
+# H420
+#
```

Comment:
fwiw, this should have an ascii-art of where the buttons are on the tablet, look at the other data files for guidance. This will help us making sure we know which button is supposed to be which one

Reply:
ascii-art added